test/functional/normalization/crio_partial_log_test.go

```diff
@@ -0,0 +1,82 @@
+package normalization
+
+import (
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+)
+//Fast test for checking reassembly logic for split log by CRI-O.
+//CRI-O split long string on parts. Part marked by 'P' letter and finished with 'F'.
+//Example:
+// 2021-03-31T12:59:28.573159188+00:00 stdout P First line of log entry
+// 2021-03-31T12:59:28.573159188+00:00 stdout P Second line of the log entry
+// 2021-03-31T12:59:28.573159188+00:00 stdout F Last line of the log entry
+//
+// Here we will emulate CRI-O split by direct writing formatted content
+var _ = Describe("Reassembly split by CRI-O logs ", func() {
+
+	var (
+		framework *functional.FluentdFunctionalFramework
+	)
+
+	BeforeEach(func() {
+		framework = functional.NewFluentdFunctionalFramework()
+		functional.NewClusterLogForwarderBuilder(framework.Forwarder).
+			FromInput(logging.InputNameApplication).
+			ToFluentForwardOutput()
+		Expect(framework.Deploy()).To(BeNil())
+	})
+	AfterEach(func() {
+		framework.Cleanup()
+	})
+
+	It("Checking correct reassembling for split logs (P,P,P,F)", func() {
+		//write partial log
+		timestamp := "2021-03-31T12:59:28.573159188+00:00"
+		msg := framework.NewCRIOLogMessage(timestamp, "May ", true)
+		err := framework.WriteMessagesToApplicationLog(msg, 1)
```

Comment:
If you dont need the error it might be cleaner as: 
```
Expect(framework.WriteMessagesToApplicationLog(msg, 1)).To(BeNil())
```
or `To(Succeed())` if `Succeed` is an option